assets/js/gf-feed-forge-admin.js

```diff
@@ -45,43 +45,75 @@ jQuery(function($) {
 
 		$('#feeds_please_wait_container').fadeIn();
 
+		gfffBatch($.toJSON(selectedFeeds), leadIds, 1000, 1, 0, null);
+	});
+
+	function gfffBatch(feeds, leadIds, size, page, count, total) {

```

Comment:
Now that the batcher is here, can you disable the "Process feeds" button as we don't want multiple batches going 😅